modules/ROOT/pages/tokenization-formats.adoc

```diff
@@ -114,6 +117,14 @@ a| * Decimal
 |*Preserve serial number sub-component* 
 |This option is for the social security data domain only.
 |Social security number
+
+|*Ignore characters*
+|Used by the masking domain to specify which characters are not masked. For example, if `Ignore characters` is set to `-` and the masking character is set to `\#`, masking applied to the number `123-45-6789` returns `\##\##-\##\#-####`.
+|Masking
+
+|*Masking character*
```

Comment:
Is it Masking Character as mentioned in Masking?

Reply:
Hmm. Would it be capitalized if I'm not referring directly to the property name in the UI?